pytorch_addons/docs/bladedisc_torch_overview.md

```diff
@@ -0,0 +1,112 @@
+## Overview
+
+![bladedisc_torch_overview](./bladedisc_torch_overview.jpg)
+
+The figure shows how a PyTorch nn.Module is compiled via BladeDISC. In short,
```

Comment:
Shall we setup a convention that anything referring to variable/function/class name in code should be quoted by \` notation ?

Reply:
+1